stdlib/src/builtin/int_literal.mojo

```diff
@@ -35,13 +35,13 @@ struct IntLiteral(Intable, Stringable, Boolable, EqualityComparable):
     )
 
     @always_inline("nodebug")
-    fn __init__() -> Self:
+    fn __init__(inout self):
         """Default constructor.
 
         Returns:
             The constructed Self object.
```

Comment:
This will need to remove the "returns" doc comment, since it is nuking the return value.